CONFIGURE.md

```diff
@@ -287,6 +341,62 @@ Another option to define a state storage is to use the following properties in t
+
+**NOTE:** RocksDB database has a limit of 4GB for the size of a database object. Due to this if you are expecting to process larger flow files than 4GB you should use the `FileSystemRepository`. The downside of using `FileSystemRepository` is that it does not have the transactional guarantees of the RocksDB repository implementation.
+
+    # in minifi.properties
+    nifi.content.repository.class.name=FileSystemRepository
+
+The Provenance Repository can be configured with the `nifi.provenance.repository.class.name` property. If not specified, it uses the `ProvenanceRepository` class by default, which persists the provenance events in a RocksDB database. Alternatively it can be configured to use a `VolatileProvenanceRepository` that keeps the state in memory and flushes upon restart, or the `NoOpRepository` to not keep track of the provenance events. By default we do not keep track of the provenance data, so `NoOpRepository` is the value specified in the default minifi.properties file.
+
+    # in minifi.properties
+    nifi.flowfile.repository.class.name=NoOpRepository
+
+
+### Configuring Volatile Repositories
+As stated before each of the repositories can be configured to be volatile (state kept in memory and flushed upon restart) or persistent. Volatile repositories have some additional options, that can be specified in the following ways:
+
+    # in minifi.properties
+    # For Volatile Repositories:
+    nifi.flowfile.repository.class.name=VolatileFlowFileRepository
+    nifi.provenance.repository.class.name=VolatileProvenanceRepository
+    nifi.content.repository.class.name=VolatileContentRepository
+
+    # configuration options
+    # maximum number of entries to keep in memory
+    nifi.volatile.repository.options.flowfile.max.count=10000
+    # maximum number of bytes to keep in memory, also limited by option above
+    nifi.volatile.repository.options.flowfile.max.bytes=1M
+
+    # maximum number of entries to keep in memory
+    nifi.volatile.repository.options.provenance.max.count=10000
+    # maximum number of bytes to keep in memory, also limited by option above
+    nifi.volatile.repository.options.provenance.max.bytes=1M
+
+    # maximum number of entries to keep in memory
+    nifi.volatile.repository.options.content.max.count=100000
+    # maximum number of bytes to keep in memory, also limited by option above
+    nifi.volatile.repository.options.content.max.bytes=1M
+    # limits locking for the content repository
+    nifi.volatile.repository.options.content.minimal.locking=true
```

Comment:
I suspect the values in this sample section used to be the default values, but they aren't any longer.  Can you change them to the current default values, please?

Reply:
I updated the values and I also found a bug in the code, where the class name of the volatile provenance repository was incorrect, so the example was not working in the production code. Fixed the issues in 4e4434732dac8f91122fc910ea067c7c752283c6